src/component/view/bootstrap/form-select.html

```diff
@@ -9,7 +9,23 @@
     multiple.bind="multiple !== false"
     autofocus.bind="autofocus"
     required.bind="required"
+    if.bind="translate !== false"
```

Comment:
Edited the second conditional. If the value is null, it will bind as if it was true (default behavior).